go.mod

```diff
@@ -13,38 +13,41 @@ require (
 	github.com/Azure/go-autorest/autorest/to v0.4.0
 	github.com/LINBIT/golinstor v0.27.0
 	github.com/aquilax/truncate v1.0.0
-	github.com/aws/aws-sdk-go v1.40.39
+	github.com/aws/aws-sdk-go v1.44.45
 	github.com/go-openapi/inflect v0.19.0
-	github.com/hashicorp/go-multierror v1.1.0
+	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/go-version v1.2.1
 	github.com/heptio/ark v1.0.0
 	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.0.0
 	github.com/kubernetes-incubator/external-storage v0.20.4-openstorage-rc7
 	github.com/kubernetes-sigs/aws-ebs-csi-driver v0.9.0
-	github.com/libopenstorage/openstorage v8.0.1-0.20211105030910-665c2f474186+incompatible
+	github.com/libopenstorage/openstorage v9.4.20+incompatible
 	github.com/libopenstorage/secrets v0.0.0-20220413195519-57d1c446c5e9
 	github.com/mitchellh/hashstructure v1.0.0
 	github.com/openshift/api v0.0.0-20210105115604-44119421ec6b
 	github.com/openshift/client-go v0.0.0-20210112165513-ebc401615f47
 	github.com/pborman/uuid v1.2.0
 	github.com/portworx/kdmp v0.4.1-0.20220710173715-5d42efc7d149
+	github.com/portworx/px-object-controller v0.0.0-20220727220448-306ddfd28652
 	github.com/portworx/sched-ops v1.20.4-rc1.0.20220401024625-dbc61a336f65
 	github.com/portworx/torpedo v0.20.4-rc1.0.20210325154352-eb81b0cdd145
 	github.com/prometheus/client_golang v1.11.0
 	github.com/sirupsen/logrus v1.8.1
 	github.com/skyrings/skyring-common v0.0.0-20160929130248-d1c0bb1cbd5e
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.7.0
+	github.com/stretchr/testify v1.7.2-0.20220317124727-77977386932a
 	github.com/urfave/cli v1.22.2
+	github.com/zoido/yag-config v0.4.0
 	gocloud.dev v0.20.0
-	golang.org/x/oauth2 v0.0.0-20201208152858-08078c50e5b5
+	golang.org/x/oauth2 v0.0.0-20220309155454-6242fa91716a
 	google.golang.org/api v0.30.0
-	google.golang.org/grpc v1.40.0
+	google.golang.org/grpc v1.43.0
+	google.golang.org/grpc/examples v0.0.0-20220715232852-f601dfac73c9 // indirect
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.21.4
+	k8s.io/api v0.24.0
```

Comment:
is this required? or can we stick to 0.21.4

Reply:
We are not updating K8s/api or K8s/apimachinery version with this commit. v0.21.4 is the version available in replace( secrtion in go mod and is overriding the version seen above.